test/integration/features/provision_various_versions.feature

```diff
@@ -1,4 +1,4 @@
-@provision-various-versions
+@provision-various-versions @openshift @minishift-only
```

Comment:
@LalatenduMohanty Does this tag ```@minishift-only``` is worth having to keep here until we manage the merge conflict volume ?
The same [question](https://github.com/minishift/minishift/pull/2061#discussion_r170177659) is asked by @budhrg in this pr #2061.

Reply:
@amitkrout The question of whether we should put the `minishift-only` tag or not, depends on what is the best way to handle changes required for downstream. It seems the easiest path forward is modify the same file for downstream. Though it might cause merge conflicts in long term, but unless we have a lot of merge conflicts it will work. 

So the answer is , we do not need the `minishift-only` tag here.